packages/protobuf-bench/report.mjs

```diff
@@ -20,38 +20,37 @@ server would usually do.
 | protobuf-javascript | ${googleProtobuf.size}  | ${googleProtobuf.minified} | ${googleProtobuf.compressed} |
 `);
 
-
 function gather(entryPoint) {
-    const bundle = build(entryPoint, false, "esm");
-    const bundleMinified = build(entryPoint, true, "esm");
-    const compressed = compress(bundleMinified);
-    return {
-        entryPoint,
-        size: formatSize(bundle.byteLength),
-        minified: formatSize(bundleMinified.byteLength),
-        compressed: formatSize(compressed.byteLength),
-    };
+  const bundle = build(entryPoint, false, "esm");
+  const bundleMinified = build(entryPoint, true, "esm");
```

Comment:
Oops. Can I ask you to add `mjs` [here](https://github.com/bufbuild/protobuf-es/blob/2657ac74842880a3871de76a708f791d73058388/Makefile#L196)?

Reply:
sure! [`2035624` (#298)](https://github.com/bufbuild/protobuf-es/pull/298/commits/2035624b20cdfc35b4b2becb09872d528378a44f)